server/src/assembly/resources/conf/logback.xml

```diff
@@ -25,14 +25,12 @@
     <statusListener class="ch.qos.logback.core.status.NopStatusListener"/>
     <appender class="ch.qos.logback.core.rolling.RollingFileAppender" name="FILEERROR">
         <file>${IOTDB_HOME}/logs/log_error.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>${IOTDB_HOME}/logs/log-error.%i.log.gz</fileNamePattern>
-            <minIndex>1</minIndex>
-            <maxIndex>5</maxIndex>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
```

Comment:
so how to limit the log size?

Reply:
is there any upper bound for each file? and the total size of files?

Reply:
OK, I'll add restrictions

Reply:
> so how to limit the log size?

I'll add restrictions

Reply:
> is there any upper bound for each file? and the total size of files?

Copy that, I'll add it